Add note to activate "Supported web addresses" in deep linking documentation (#12269)
flutter/website: src/content/cookbook/navigation/set-up-app-links.md

directly through web browser or another app.
One way is to create a Google Doc, add the link, and tap on it.

:::note

> You may need to enable the toggle for "Supported web addresses" manually
> in your Android app's settings if you are debugging locally, and not
> downloading the app from the Play Store.

Reviewer: ```suggestion
If you are debugging locally (and not downloading the app from the Play Store),
you might need to enable the toggle for **Supported web addresses** manually.
```
